webview/src/experiments/util/columns.ts

```diff
@@ -80,3 +84,8 @@ export const leafColumnIds = (
 
 export const isExperimentColumn = (id: string): boolean =>
   id === EXPERIMENT_COLUMN_ID
+
+export const isDefaultColumn = (id: string) =>
```

Comment:
> [Q] Couldn't we just use DEFAULT_COLUMN_IDS instead of importing all of the individual IDs?

Yes, thanks for catching that! 

> [Q] Won't the addition of these two columns break <WithExpColumnNeedsShadowUpdates>?

Forgot to mention this in a review comment, For now, the columns aren't sticky:

https://github.com/iterative/vscode-dvc/assets/43496356/10adf0f9-5f48-411f-b1b5-1d00449eb98c

I wanted to save it for a follow-up since I was having some difficulty making three columns sticky and have the experiment column conditionally have a shadow.